modules/extdata/uri/META-INF/ignite.xml

```diff
@@ -33,5 +33,6 @@
     -->
     <util:list id="tasks">
         <value>org.apache.ignite.spi.deployment.uri.tasks.GarHelloWorldTask</value>
+        <value>org.apache.ignite.spi.deployment.uri.tasks.SimpleNamedTask</value>
     </util:list>
 </beans>
```

Comment:
Should there be a newline?

Reply:
Maybe, but it wasn't before this change, so it's probably not a big deal